Salwador/DevEnt.geojson

```diff
@@ -0,0 +1,5 @@
+{
+"apikey" : "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",
+"moderationURL": "https://api.openai.com/v1/moderations",
+"imageGeneratorURL" : "",

```

Comment:
Do you use this geojson somewhere? I wouldn't checkin the API key.

Reply:
Removed that file, I thought it works like JSON file like in react. :)